pkg/runtime/k0s/utils.go

```diff
@@ -74,3 +78,22 @@ func FetchKubeconfigAndGetKubectl(ssh ssh.Interface, host net.IP, rootfs string)
 	}
 	return nil
 }
+
+func (k *Runtime) WaitK0sReady(ssh ssh.Interface, host net.IP) error {
+	times := DelayTimes
+	for {
+		times--
+		if times == 0 {
+			break
+		}
+		time.Sleep(time.Second * 2)
+		bytes, err := ssh.Cmd(host, "k0s status")
+		if err != nil {
+			return err
+		}
+		if strings.Contains(string(bytes), "Process") {
+			return nil
+		}
+	}
+	return errors.New("failed to start k0s")
```

Comment:
Please make the error more readable. 
`failed to start k0s: failed to get k0s status after 10 seconds` ?

Reply:
sounds good, Thx!